news/2022-07-31-the-sound-design-of-lazer.md

```diff
@@ -12,13 +12,11 @@ Sound design relates to how any particular sound can make its listener *feel*. T
+nekodex is a full-stack engineer born in Australia who joined the osu! team as a web developer in 2015 after peppy asked if he was interested in helping rebuild the osu! website. nekodex has been with the team ever since, working on various osu!-related things, such as the review system for beatmap modding, the chat system etc., but today he describes himself as a hybrid audio designer/developer and isn't too involved in the web side of things anymore. 
 
-nekodex as an alias came to be in 2012, as Jamie had a musical background while growing up, played with early music software, learned from YouTube tutorials and eventually started messing around in DAWs (digital audio workstations) until music came out. As Jamie was getting serious with music production, peppy was preparing osu!stream (an iOS spinoff of osu! that was created in 2010) for the public view, which required original music for the game.
+His first involvement with osu! as a music producer would come in 2011 as peppy was preparing osu!stream (a spinoff of osu! for iOS) and needed original music for the game. Since peppy and nekodex are real life friends and peppy knew about his music, he got the chance to create the [main menu theme](https://soundcloud.com/nekodex/osu-stream-theme) and [a track](https://soundcloud.com/nekodex/liquid-future) for the release of osu!stream, and later on went on to create several official songs for osu!(stable) throughout the years that we all know and love, such as [welcome to osu!](https://soundcloud.com/nekodex/welcome-to-osu), [circles!](https://soundcloud.com/nekodex/circles) and [aureole](https://soundcloud.com/nekodex/aureole).
 
-Since peppy and Jamie are real life friends and peppy knew about his music, he got the chance to create some themes and tracks for osu!stream, and later on got some of his own songs mapped for it as well. He then later went on to create several official songs throughout the years that we all know and love, such as [welcome to osu!](https://soundcloud.com/nekodex/welcome-to-osu), [circles!](https://soundcloud.com/nekodex/circles) and [aureole](https://soundcloud.com/nekodex/aureole).
-
-When peppy decided to start working on a new client from the ground up in 2016 (after trying to make lazer within the stable client and figuring out such a thing would not be feasible), he wanted to not only make the game look good, but also *feel* good. That task required an audio designer, and with nekodex already having done several amazing works for osu!, peppy approached him.
+In 2021 during development of osu!(lazer) and peppy saw that the sound design was lacking compared to all of the new visuals and asked nekodex if he was willing to try and give it a facelift, at that point his primary work such as the osu! website and lazer were in a good enough spot so he decided to accept the task and committed to it fully. 
```

Comment:
I guess I was unclear... my primary work was on the osu! website, not lazer.

I was trying to say that peppy and the team had gotten lazer to the point where the architecture had settled enough for me to start working on adding SFX.

Reply:
I will fix this once I am back home from vacation tomorrow